test/compile/mark/bar.test.ts

```diff
@@ -100,4 +100,40 @@ describe('Mark: Bar', function() {
       });
     });
   });
+  
+  describe('ranged bar', function() {
+    it('vertical bars should work with aggregate', function() {
```

Comment:
Incorrect description of what you're testing:  'vertical bars should work with aggregate'